lib/output.js

```diff
@@ -139,13 +140,19 @@ function inlineSourceMap (output) {
 }
 
 exports.writeOutputs = function(opts, outputs, baseURL) {
-  // remove 'file:' part
-  var basePath = baseURL.substr(5);
+  var basePath = filePath(baseURL);
+  var outputPath;
 
-  if (opts.outFile)
+  if (opts.outFile && basePath) {
     opts.outFile = path.relative(basePath, path.resolve(opts.outFile));
+    outputPath = path.dirname(path.resolve(basePath, opts.outFile)) + path.sep;
```

Comment:
Don't see any reason to add the trailing separator - only used with `path.relative`, which doesn't care.